javascript/browser-apis/reading-writing/blobs.md

```diff
@@ -33,34 +20,44 @@ links:
 ---
 ## Content
 
-A *blob* object represents an "almost file" object of raw data. It represents data that isn't in JavaScript's standard format. You use *FileReader* to read data from a blob.
+A binary large object, or *blob*, represents a file-like object of immutable raw data. It represents data that isn't in JavaScript's standard format. You use `FileReader` to read data from a blob.
 
 To create a blob, use the blob constructor:
 ```javascript
-var debug = {hello: "world"};
-var blob = new Blob([
-  JSON.stringify(debug, null, 2)
-], {type : 'application/json'});
+let debug = {
+  hello: "world",
+};
+
+let blob = new Blob(
+  [JSON.stringify(debug, null, 2)],
```

Comment:
why are we first stringifying the debug object? 

I'd say try to go over everything that's happening in this code snippet, line by line, underneath the example

Reply:
why the array ? why the stringify if we specify "application/json" down below? why the `null` and `2` arguments (I know they are used too prettify the stringification, but they might confuse the uesrs)